test/stubby_test.exs

```diff
@@ -58,4 +68,17 @@ defmodule StubbyTest do
       assert FakeStub.fake_function("anything") == "works!"
     end
   end
+
+  describe "module based function generation" do
+    setup do
+      TestModuleStub.setup
+      :ok
```

Comment:
Maybe a different issue, but can `.setup/0` return `:ok`  so this turns into a one-liner? I'm sure there may be additional setup but it hurts a little to _always_ need to specify `:ok` when the setup is ok.  Or is there a reason we might want to use the return value of `:ets.new/2` here?